draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1641,6 +1641,22 @@ corresponding curve.
 See {{security-considerations-targets}} for more details, and
 {{new-suite}} for guidelines on choosing k for a given curve.
 
+The hash\_to\_field function described below may not be most efficient
+for all fields. For example, when hashing to an element of the extension
```

Comment:
```suggestion
## Efficiency considerations in extension fields {#hashtofield-exteff}

The hash\_to\_field function described in this section is inefficient for certain
extension fields. Specifically, when hashing to an element of the extension
```

Reply:
I think it might be better to add a separate subsection because otherwise I worry that this info gets lost...

Reply:
That works for me!